website/content/en/highlights/2022-05-03-0-22-0-upgrade-guide.md

```diff
@@ -0,0 +1,42 @@
+---
+date: "2022-05-03"
+title: "0.22 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.22.0"
+authors: ["spencergilbert"]
+pr_numbers: [11714]
+release: "0.22.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.22.0 release includes **breaking changes**:
+
+1. [`kubernetes_logs` source rewritten to use `kube-rs`](#kubernetes-logs)
+
+We cover them below to help you upgrade quickly:
+
+## Upgrade guide
+
+### Breaking changes
+
+#### `kubernetes_logs` source rewritten to use `kube-rs` {#kubernetes-logs}
+
+Our `kubernetes_logs` source has been updated to use [`kube-rs`](https://kube.rs/)
+as the foundations of our integration. `kube-rs` is a CNCF Sandbox Project and
+has served as the basis of a number of applications that work with the Kubernetes API.
+We made this change to improve the reliability and stability of our `kubernetes_logs`
+source and to ensure our compatibility with the Kubernetes API as it evolves.
+
+There are two user facing changes for this update: the `list` verb is now required
+for Vector's ClusterRole, and the [`proxy`](https://vector.dev/docs/reference/configuration/global-options/#proxy)
+options are no longer used for the client.
+
+The [`vector` Helm chart](https://github.com/vectordotdev/helm-charts/tree/develop/charts/vector)
+has been updated to include this new verb as of `0.7.0`, and the Kustomize based
+manifests in the `vector` repo have been updated as part of this release. If you
+are managing your own manifests separately you need to ensure the ClusterRole used
+by Vector includes both the `list` and `watch` verbs for `pods` and `namespaces`.
```

Comment:
I think I'd like to start getting in the habit of writing these highlights for "busy users". That is, get right to the point, and then provide additional details for the interested.

For this one, this might look something like:

> The `kubernetes_logs` source has had two breaking changes:
> 
> * It now requires the `list` verb for Vector's ClusterRole. If you are using the Helm chart, version 0.7.0 includes this change. Otherwise, make sure to add it to your manifest.
> * The `proxy` configuration was dropped. Instead, configure any needed proxy configuration in your `kubeconfig`

Then follow with the motivation and details.

I got thinking about this again after I saw Gitlab including a "what you need to know" in a recent announcement: https://about.gitlab.com/blog/2022/03/24/efficient-free-tier/

Reply:
This could also probably make 0.21.0 if it was merged within the next week.